### PR TITLE
Fix/97667 reading options

### PR DESCRIPTION
**Pull request by mitogh**
Ticket: https://central.tri.be/issues/100832

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [Front_Page_View.php:93](https://github.com/moderntribe/the-events-calendar/blob/3bc8462c20c7d725922dbb6c36ed3989d8a74c25/src/Tribe/Front_Page_View.php#L93)
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [Front_Page_View.php:117](https://github.com/moderntribe/the-events-calendar/blob/3bc8462c20c7d725922dbb6c36ed3989d8a74c25/src/Tribe/Front_Page_View.php#L117)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Front_Page_View.php [hunk 1]
```diff
@@ -69,6 +72,70 @@ public function parse_query( WP_Query $query ) {
...
+	 */
+	public function parse_customizer_query( $query ) {
+
+		$data = tribe_get_request_var( 'customized', array() );
+
+		if ( empty( $data ) ) {
+			return;
+		}
+
+		$data = json_decode( wp_unslash( $_REQUEST['customized'] ), true );
```
Comment (Camwyn) [inline src/Tribe/Front_Page_View.php]: I'm not familiar with `tribe_get_request_var` but is there some reason we can't do `json_decode( wp_unslash( $data ) )` here (or above)? 

Why the second step?

Comment (mitogh) [reply src/Tribe/Front_Page_View.php]: You're right! we don't need it anymore as the value is on `$data`. Thanks  Sending the fix

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [Front_Page_View.php:93](https://github.com/moderntribe/the-events-calendar/blob/a33b50c00a3ca78e691b43f883a006835d4ea5b1/src/Tribe/Front_Page_View.php#L93)
:x: | empty() and isset() must only contain variables (PHP 5.2 compatibility) | [Front_Page_View.php:117](https://github.com/moderntribe/the-events-calendar/blob/a33b50c00a3ca78e691b43f883a006835d4ea5b1/src/Tribe/Front_Page_View.php#L117)

> via jenkins/codesniffer-phpcs

**Pull request merged**